sycl/test/regression/macro_conflict.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clangxx -fsycl %s -o %t.out
```

Comment:
If you only want to check compilation, I suggest to add `-fsyntax-only -Xclang -verify` options

Reply:
Thanks, I will fix as you suggested.